signalr-go-server/pkg/signalr/jsonhubprotocol.go

```diff
@@ -26,14 +27,11 @@ func (j *JsonHubProtocol) UnmarshalArgument(argument interface{}, value interfac
 func (j *JsonHubProtocol) ReadMessage(buf *bytes.Buffer) (interface{}, bool, error) {
 	data, err := parseTextMessageFormat(buf)
 	switch {
-	case err == io.EOF:
+	case errors.Is(err, io.EOF):
```

Comment:
? hasn't seen this before. What's the difference?

Reply:
Errors might be wrapped to give further information (instead of writing to Exception.Data in .Net). In this case, == fails. Is() tests if the wrapped root cause has this value

Reply:
Errors can be wrapped to add additional information (like Exception.Data in .Net). On wrapped errors == will fail but Is() does still succeed